Documentation/Extensions/RecommendedExtensions.rst

```diff
@@ -0,0 +1,65 @@
+.. include:: /Includes.rst.txt
+
+.. _recommended-extensions:
+
+======================
+Recommended Extensions
+======================
+
+Here's a selection of recommended extensions that provide essential features,
+streamlined workflows, and powerful enhancements for TYPO3 development.
+
+*   TYPO3 console: `helhum/typo3-console <https://extensions.typo3.org/extension/typo3_console>`__
```

Comment:
I believe we have a `ter` directive or something like that to use here for linking? @linawolf surely remembers? :)

Reply:
Should be

    :t3ext:`typo3_console`

as I remember correctly.

Reply:
Thank you :) I didn't know that. :)